detectable/src/main/java/com/synopsys/integration/detectable/detectables/gradle/inspection/GradleInspectorExtractor.java

```diff
@@ -73,6 +74,13 @@ public Extraction extract(final File directory, final File gradleExe, String gra
             arguments.add(String.format("-DGRADLEEXTRACTIONDIR=%s", outputDirectory.getCanonicalPath()));
             arguments.add("--info");
 
+            if (proxyInfo.getHost().isPresent()) {
+                arguments.add("--Dhttps.proxyHost=" + proxyInfo.getHost());
+            }
+            if (proxyInfo.getPort() != 0) {
+                arguments.add("--Dhttps.proxyPort=" + proxyInfo.getPort());
+            }
+
```

Comment:
The current workaround for the customer is to add these arguments to the custom gradle build command. Will supplying multiple of the same arguments cause a problem with gradle? If so should we attempt to de-duplicate the argument the way we do the "dependencies" argument.